frontend/templates/product-list/MetaTags.tsx

```diff
@@ -88,17 +77,15 @@ export function MetaTags({ productList }: MetaTagsProps) {
                <link rel="canonical" href={canonicalUrl} />
                <meta
                   name="description"
-                  content={
-                     productList.metaDescription || productList.description
-                  }
+                  content={productList.metaDescription ?? undefined}
                />
             </>
          )}
-         <title>{title}</title>
-         <meta property="og:title" content={title} />
+         <title>{metaTitle}</title>
+         <meta property="og:title" content={metaTitle} />
          <meta
             name="og:description"
-            content={productList.metaDescription || productList.description}
+            content={productList.metaDescription ?? undefined}
```

Comment:
This seems odd. Is this because we're explicitly handling the metaDescription -> description fallback elsewhere? In that case, it seems like it'd make more sense to conditionalize the meta tag (and the one on line 80) entirely rather than include the tag with an empty description.

Reply:
its because we convert everything into strings or null and content only takes strings or undefined, not null.